### DISCUSSION TITLE
How to extend the $PATH environment variable

### DISCUSSION THREAD

**Discussion by ConnorThomp** [mKeRix/room-assistant#438] [Q&A]
So I am a total noob when it comes to this sorta stuff, and was hoping someone would be able to help me out.

I am confused by this step: 
_To make the commands we install with npm available the $PATH environment variable needs to be extended as well. Edit the file ~/.profile (e.g. with nano ~/.profile) and add the PATH="$PATH:/opt/nodejs/bin" to the end of the file. Save, then run source ~/.profile._

Where am I meant to put the extra line of code? Any help would be appreciated, thank you! :)

**Comment by mKeRix** [comment]
In the home folder of the user that you are using should be a hidden file called `.profile`, which contains a script. On a Raspberry Pi you would usually find this file at `/home/pi/.profile`. The command `nano ~/.profile` will open the file in the editor on any Linux system though. Then just scroll all the way to the end of the file, paste the additional line and save. :)
